### ISSUE TITLE
Retrieving team data from a season in which there are none should return an error

### ISSUE THREAD

**Issue by tomwarrens** [rkaahean/xgpy#2]
Hi, 

right now, if you look for team data in a season in which it didn't play in one of the top leagues you still get a result. 

For instance, Frosinone lastly player in Serie_A in 2018/2019, but still you don't get any errors with the following code: 

`team = UnderstatTeam('Frosinone', 'Serie_A')`

`team_players = (pd.DataFrame(team.get_team_player_summary(season = 2020)))`

`display(team_players)`

![image](https://user-images.githubusercontent.com/21237959/105861751-57375a80-5fef-11eb-93b0-4c4599fc12bd.png)

**Comment by rkaahean** [comment]
@tomwarrens good catch. Probably need a way to validate the season before actually fetching the data. 

Currently on understat, 
https://understat.com/team/Frosinone/2020 - gives the data from 2018/19 season as well.
